packages/quantic/force-app/main/default/lwc/quanticSearchBox/quanticSearchBox.html

```diff
@@ -40,7 +79,7 @@
                     <div class="slds-media slds-listbox__option slds-listbox__option_plain slds-media_small" role="option">
                       <span class="slds-media__figure slds-listbox__option-icon"></span>
                       <span class="slds-media__body">
-                        <span class="slds-truncate" title={suggestion.value}>{suggestion.value}</span>
+                        <lightning-formatted-rich-text class="slds-truncate" value={suggestion.value}></lightning-formatted-rich-text>
```

Comment:
Why this change?

Reply:
for highlighted substrings in query suggestions